mock/breez-sdk/src/lib.rs

```diff
@@ -648,9 +696,23 @@ pub async fn parse(input: &str) -> Result<InputType> {
         return Ok(Bolt11 { invoice });
     }
 
-    // If the input is not a Bolt11, we just assume it is a LNURL-withdraw request.
-    // This is being used for topups, LNURL-pay is less used.
     // Without requesting the server, it is not possible to know whether an LNURL string is a pay or withdraw request
+    // So instead we interpret the string 'lnurlp' as if it was an LNUrL-pay string
+    if input == "lnurlp" {
+        println!("Returning LnUrlPay");
+        return Ok(InputType::LnUrlPay {
+            data: LnUrlPayRequestData {
+                callback: "https://lnurl.dummy.com/lnurl-pay/callback/e9a0f330f34ac16d297094f568060d267bac6319a7f0d06eaf89d7fc1512f39a".to_string(),
+                min_sendable: 0,
```

Comment:
Min min_sendable is 1 according to spec
```suggestion
                min_sendable: 1,
```